core/src/test/java/io/questdb/test/cairo/TableReaderTest.java

```diff
@@ -2908,6 +2908,42 @@ public void testRemovePartitionByYearReload() throws Exception {
         testRemovePartitionReload(PartitionBy.YEAR, "2020", 3000, current -> Timestamps.addYear(Timestamps.floorYYYY(current), 1));
     }
 
```

Comment:
Is this test related to the change ? It passes even when I revert the change .

Reply:
This test was supposed to be for #3582